libs/traits/src/swaps.rs

```diff
@@ -112,25 +118,33 @@ pub struct SwapState<AmountIn, AmountOut, Currency> {
 	/// Amount of incoming currency already swapped denominated in outgoing
 	/// currency
 	pub swapped_out: AmountOut,
+
+	/// Ratio used to swap `swapped_out` into `swapped_in`
+	pub ratio: Ratio,
```

Comment:
The `OrderInfo` returned by `TokenSwaps::get_order_details` contains a `ratio` field of type `OrderRatio<Ratio>`. Maybe that suffices such that we don't need to introduce an additional ratio field?

Reply:
Thanks for the explanation! Makes sense to keep as is then.